lib/svg-sprite/utils/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Checks if value is a callable function.
+ *
+ * @param {string} value The value to check.
+ * @returns {boolean}    Returns true if value is correctly classified, else false.
+ */
+function isFunction(value) {
+    if (value && typeof value === 'function') {
+        return true;
+    }
+
+    return false;
+}
+
+/**
+ * Checks if value is the language type of Object. (e.g. arrays, functions, objects, regexes, new Number(0),
+ * and new String(''))
+ *
+ * @param {any} value The value to check.
+ * @returns {boolean} Returns true if value is an object, else false.
+ */
+function isObject(value) {
```

Comment:
Let's go with this, I think it's safer:

```js
function isObject(value) {
  return typeof value === 'object' && !Array.isArray(value) && value !== null;
}
```

Reply:
this change could break existing code (outcome of `writeFiles` is altered), including lib tests failing with a change. The checks does not work for arrays now for example. It's very terse and different from original lodash implementation

Reply:
Weird, that's what I've been using for years myself with vanilla JS. I still don't think `isObject` should return true for Array or Function, but I haven't checked the lodash implementation.

Reply:
OK, I had a closer look at the test failures and I think we should fix our tests instead. It seems we are passing an Array in writeFiles in some cases which is causing the test failures, otherwise my suggestion should be OK. @Kreeg what's your take on this?

Reply:
@XhmikosR it is better to change. I'll take a look

Reply:
@XhmikosR I see you already land these changes. That's good